### ISSUE TITLE
Wildcard certificates

### ISSUE THREAD

**Issue by patrickbaber** [Daanra/laravel-lets-encrypt#30]
Can wildcard certificates be issued for domains like *.example.org? Let's Encrypt needs a DNS validation for this. I know this depends on the DNS provider, but it would be nice if the package had methods to get the string for the TXT record.

**Comment by Daanra** [comment]
I agree with you that this feature would be great. But as you said, to make this work automatically, we'd need different implementations for the DNS providers. 

I have considered adding a base class that could be implemented by the user to take care of uploading the DNS record to their DNS provider, and then this package would take care of the rest. I wasn't quite sure yet if it was worth the effort, as not a lot of users actually need it.

If you're willing to submit a PR then I'd be happy to merge it.
